docs/source/reference/services.md

```diff
@@ -346,7 +346,7 @@ and taking note of the following process:
         headers = {
             'Authorization' : 'token %s' % api_token,
         },
-    )
+    ))
```

Comment:
I think this code snippet is wrong, but I think the issue the surplus parenthesis after `.join(`, so instead of adding one, lets remove one!

Reply:
Ah yes !! 😅